### PR TITLE
[mariadb] Add secrets resolving for MariaDB passwords

### PR DESCRIPTION
**Pull request by s10**
Additionally add single quotation to support usage of all special symbols

### CODE REVIEW

File: common/mariadb/Chart.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
 description: A Helm chart for Kubernetes
 name: mariadb
-version: 0.12.1
+version: 0.13.0
```
Comment (businessbean) [inline common/mariadb/Chart.yaml:4]: The version is already used [in this PR](https://github.com/sapcc/helm-charts/pull/6877/files#diff-8d953323a432c852ed44578d9c4fc78801db45169ed17cbcd0af91128fe4eb14R4)

**Review by businessbean — CHANGES_REQUESTED** [review]
lgtm, just bump the version again...

File: common/mariadb/Chart.yaml (hunk 1 above)
Comment (s10) [reply common/mariadb/Chart.yaml]: Updated to 0.14.0

**Review by galkindmitrii — APPROVED** [review]

**Review by businessbean — APPROVED** [review]
lgtm

**Pull request merged**